application-task-ui/src/main/resources/TaskManager/TaskManagerClass.xml

```diff
@@ -309,11 +320,12 @@
         &lt;/div&gt;
       &lt;/div&gt;
   #else
-    &lt;input class="form-control" id="TaskManager.TaskManagerClass_0_progress" name="TaskManager.TaskManagerClass_0_progress" type="number" value="${value}" /&gt;
+    &lt;input class="form-control" id="TaskManager.TaskManagerClass_0_progress" name="TaskManager.TaskManagerClass_0_progress" type="number" min="0" oninput="validity.valid||(value='');" value="${value}" /&gt;
```

Comment:
We could also set the upper bound to be 100. 
```suggestion
    &lt;input class="form-control" id="TaskManager.TaskManagerClass_0_progress" name="TaskManager.TaskManagerClass_0_progress" type="number" min="0" max="100" oninput="validity.valid||(value='');" value="${value}" /&gt;
```

Reply:
okay, then I'll close my question (issue) #65